Source/JavaScriptCore/inspector/scripts/tests/expected/commands-with-async-attribute.json-result

```diff
@@ -336,7 +336,7 @@ void DatabaseBackendDispatcher::executeSQLSyncOptionalReturnValues(long protocol
     if (!!out_opt_columnNames)
         protocol_jsonMessage->setArray("columnNames"_s, out_opt_columnNames.releaseNonNull());
     if (!!out_opt_notes)
-        protocol_jsonMessage->setString("notes"_s, out_opt_notes);
+        protocol_jsonMessage->setString("notes"_s, WTFMove(out_opt_notes));
```

Comment:
just to make sure, this wont steal the value from the command handler, right?  like if the value returned is something still in use elsewhere (e.g. an identifier or some DOM attribute or etc.) this won't cause a use-after-move?